### ISSUE TITLE
windows installer?

### ISSUE THREAD

**Issue by psacc** [thorstenb/odpdown#44]
sad to ask about this, is there in the roadmap an easy self contained installation procedure for windows (ie. a chocolatey.org package) with dependecies and python runtime management?

thanks

**Comment by thorstenb** [comment]
not really, never did that - can you provide pointers, howtos - or perhaps ideally, contribute that yourself?

Other than that, what *is* on my list of things to eventually do is turning this into a LibreOffice extension.

**Comment by psacc** [comment]
The only pointer I can provide right now is https://chocolatey.org/docs :)

I like the LibreOffice extension idea, but could it still be run from a cli? If that's the case LibreOffice integration could be the best solution.

Thanks!

**Comment by thorstenb** [comment]
Hmm. Quite a mouthful to read, so volunteers with experience appreciated. :)

To your question, no, not easily. It would be a GUI, with a file selection & ending up with slides in Impress
